Cython/Compiler/ModuleNode.py

```diff
@@ -2190,81 +2188,153 @@ def generate_clear_function(self, scope, code, cclass_entry):
         code.putln("}")
         code.exit_cfunc_scope()
 
-    def generate_getitem_int_function(self, scope, code):
-        # This function is put into the sq_item slot when
-        # a __getitem__ method is present. It converts its
-        # argument to a Python integer and calls mp_subscript.
+    def generate_getitem_function(self, scope, code):
+        # Implement both 'sq_item()' and 'mp_subscript()'.
+        # Use a delegating wrapper for the less suitable slot.
```

Comment:
I don't think this comment is (quite) true any more (at least it doesn't do that for `mp_subscript`)